website_chd_product_configurator/templates/templates.xml

```diff
@@ -0,0 +1,369 @@
+ <openerp><data>
+
+
+  <template id="assets_frontend" inherit_id="website.assets_frontend" name="chd_product_configurator">
+  <xpath expr="." position="inside">
+      <script type="text/javascript" src="/website_chd_product_configurator/static/src/js/website_chd_product_configurator.js"></script>
+      <link rel='stylesheet' href='/website_chd_product_configurator/static/src/css/website_chd_product_configurator.css'/>
+  </xpath>
+  </template>
+
+
+  <template id="conf_start">
+    <t t-call="website.layout">
+      <t t-set="title">Product Configurator</t>
+      <div class="oe_structure"/>
+      <div class="oe_structure">
+        <div class="container">
+        <h3>configurable products available</h3>
+         <form t-attf-action="/chd_init/" method="post" >
+            <select t-attf-name="selected_id" class= "form-control product_selection" id="prod_select_id">
+                <option></option>
+                <t t-foreach="conf_products" t-as="conf_product">
+                    <option t-att-value="conf_product.id" class= "form-control product_selection">
+                         <t t-esc="conf_product.name"/>
+                    </option>
+                </t>
+           </select>
+        <button type="submit" class="btn btn-primary btn-lg pull-center" >Select Configurable Product</button>
+        </form>
+        <div class="clearfix"/>
+        </div>
+      </div>
+      <div class="oe_structure"/>
+    </t>
+  </template>
+
+ <template id="configurator">
+    <t t-call="website.layout">
+       <t t-set="title">Product Configurator</t>
+          <div class="oe_structure"/>
+          <div class="oe_structure">
+          <div class="container">
+            <h3 id="c_title">Configuration options for <t t-esc="curr_product_id.name" /></h3>
+            <form id="mainf" t-attf-action="/chd_init/#{ curr_product_id.id }/" method="post"  enctype="multipart/form-data"  class="form-horizontal">
+            <div class="row">
+                <div class="col-sm-9">
+                    <div class="row">
+                         <div class="col-sm-1" >
+                            <label for="quantity" >Quantity</label>
+                         </div>
+                         <div class="col-sm-2" >
+                            <input required="true" type="number"  min="0" name="quantity" class="form-control input-sm"/>
```

Comment:
boolean attributes in HTML5 don't have "true" as value, but either simply exist (without value) or have their own name as value: http://www.w3.org/TR/2008/WD-html5-20080610/semantics.html (3.2.2), so this would be `required="required"`. Given that you use HTML5 anyways, you might also set `type="number"`